mesheryctl/pkg/utils/scripts.go

```diff
@@ -162,3 +162,44 @@ func GenerateConfigAKS(resourceGroup, clusterName string) error {
 
 	return generateCFG.Run()
 }
+
+
+// GenerateConfigEKS generates kube config file in /tmp/meshery/kubeconfig.yaml for a GKE cluster
+func GenerateConfigEKS(region, cluster string) error {
+	script := fmt.Sprintf(`
+	set -e
+	set -o pipefail
+
+	TARGET_FOLDER="~/.meshery/"
```

Comment:
@hexxdump can we avoid creating a new directory? We can reuse the user's existing `MesheryFolder` (https://github.com/layer5io/meshery/blob/master/mesheryctl/pkg/utils/helpers.go#L63).

Reply:
This portion of the command can be done in golang. Let's put as little as possible into shell script.